core/templates/services/exploration-features.service.ts

```diff
@@ -17,25 +17,43 @@
  *               the exploration editor.
  */
 
-import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+export interface IExplorationDataDict {
+  'param_changes': IParamChanges[] | [];
```

Comment:
I think you meant `||` here and below.

Reply:
hi @kevintab95 
I googled it again and found out that it was correct 
 you can check it  under the [Union Type and Interface](https://www.tutorialspoint.com/typescript/typescript_interfaces.html) section

Reply:
Thanks for the explanation @abhayraghuwanshi!